crates/cli-support/src/js/mod.rs

```diff
@@ -3143,6 +3143,38 @@ impl<'a> Context<'a> {
                 assert!(!variadic);
                 self.invoke_intrinsic(intrinsic, args, prelude)
             }
+
+            AuxImport::LinkTo(path, content) => {
+                assert!(kind == AdapterJsImportKind::Normal);
+                assert!(!variadic);
+                assert_eq!(args.len(), 0);
+                if self.config.allow_links {
+                    let base = match self.config.mode {
+                        OutputMode::Web
+                        | OutputMode::Bundler { .. }
+                        | OutputMode::Deno
+                        | OutputMode::Node {
+                            experimental_modules: true,
+                        } => "import.meta.url",
+                        OutputMode::Node {
+                            experimental_modules: false,
+                        } => "require('url').pathToFileURL(__filename)",
+                        OutputMode::NoModules { .. } => "script_src",
+                    };
+                    Ok(format!("new URL('{}', {}).toString()", path, base))
+                } else {
+                    if let Some(content) = content {
+                        Ok(format!(
+                            "\"data:application/javascript,\" + encodeURIComponent(`{}`)",
+                            content.replace('`', "\\`")
+                        ))
+                    } else {
+                        Err(anyhow!("wasm-bindgen needs to be invoked with `--allow-links`, because \"{}\" cannot be embedded.\n\
+                             `--allow-links` is safe with webpack 5 or no bundler at all.\n\
+                             For other bundlers, ensure they support the `new URL('…', import.meta.url)` syntax.", path))
```

Comment:
I don't know if 'ensure they support' is the right phrasing here. Often the solution would instead be to install a plugin which adds support, or to somehow tell the bundler to copy all the linked modules into the output directory where they're expected to be relative to the shim.